src/cli.ts

```diff
@@ -53,6 +53,10 @@ program
     "Heading delimiter to use in place of '::'.",
     "::"
   )
+  .option(
+    "--replace-heading",
+    "Replace the heading as well as its content."
+  )
```

Comment:
It seems a little bit surprising to me that this would be a global option for the patch applier when that particular option is only valid for certain kinds of patch instructions; what was your intention here?